drivers/interrupt_controller/intc_gic.c

```diff
@@ -220,11 +220,7 @@ static void gic_cpu_init(void)
 }
 
 /**
- *
  * @brief Initialize the GIC device driver
- *
- *
- * @return N/A
  */
```

Comment:
This block should probably be placed right before the function `arm_gic_init()` (new line 228) and not before the defines.